kbncontent.go

```diff
@@ -111,6 +111,46 @@ func (v VisualizationDescriptor) SemanticType() string {
 	}
 }
 
+// CanUseFilter returns true if the visualization makes queries.
+func (v VisualizationDescriptor) CanUseFilter() bool {
+	switch v.SavedObjectType {
+	case "search":
+		return false
+	case "visualization":
+		if v.isTSVB() {
+			switch v.TSVBType() {
+			case "markdown":
```

Comment:
TSVB markdown charts _can_ make queries. They don't always, but they can.

Reply:
Is there a way we can check if the TSVB markdown is doing queries?

Reply:
As far as I can tell, the best way would be to look through the markdown content for mustache syntax `{{ something }}`. You might also be able to check if there are any variables defined in the `params` section. 

If you are planning on parsing the `visState` to find this info, the @elastic/kibana-visualizations team would be your best bet for help.